homeassistant/components/daikin/switch.py

```diff
@@ -30,6 +33,11 @@ async def async_setup_entry(hass, entry, async_add_entities):
                 if zone != ("-", "0")
             ]
         )
+    if daikin_api.device.support_advanced_modes:
+        # It isn't possible to find out from the API responses if a specific
+        # device supports the streamer, so assume so if it does support
+        # advanced modes.
+        async_add_entities([DaikinStreamerSwitch(daikin_api)])
```

Comment:
Please make a single call to `async_add_entities`. Ie, collect all potential entities first in a list, then if we have any entities we add them with `async_add_entities`.